### DISCUSSION TITLE
Can SE separate left-right audio for its functions?

### DISCUSSION THREAD

**Discussion by coastal45** [SubtitleEdit/subtitleedit#6741] [General]
Specifically, I have an audio track where the channels contain different languages, and I need a single channel to pass to whisper.
Can SE do this, or perhaps some other way to do this?

**Comment by niksedk** [comment]
Sorry, this is pretty advanced, so find a good audio editor with all kind of filters/mixers/muxers and got nuts :)

SE does have the "Use front center" option for 5.1/7.1/9.1 audio (lower right corner):
![image](https://user-images.githubusercontent.com/1261331/224568772-ee22b54c-c531-41bc-8260-f33081b583b3.png)

**Comment by coastal45** [comment]
I had the same experience. Odd how something so easy in the analog world can become so difficult. Video software I know of doesn't support this.

But ffmpeg may support this. Check the answers here - [Ffmpeg Audio Stereo to Mono using only left channel](https://superuser.com/questions/702552/ffmpeg-audio-stereo-to-mono-using-only-left-channel)
I'm not familiar with using ffmpeg by command line, but since SE uses it maybe something is possible.

**Comment by coastal45** [comment]
Ultimately I used the ffmpeg command from the link above -
`ffmpeg -i video.mp4 -map_channel 0.1.0 -map_channel 0.1.0 -c:v copy stereo.mp4`

and it worked correctly.
